common/src/main/java/org/jboss/hal/testsuite/creaper/command/logging/AddLogCategory.java

```diff
@@ -0,0 +1,93 @@
+package org.jboss.hal.testsuite.creaper.command.logging;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+public class AddLogCategory extends ManipulateLogCategory {
+
+    private boolean replaceExisting;
+
+    protected AddLogCategory(Builder builder) {
+        setBaseProperties(builder);
+        replaceExisting = builder.replaceExisting;
+    }
+
+    @Override
+    public void apply(OfflineCommandContext ctx) throws Exception {
```

Comment:
Do we need this to be implemented at all or should we rather throw something like `UnsupportedOperationException` instead?